spec/italian_food_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'italian_menu'
+
+describe ItalianMenu do
+  subject(:italian_menu) {described_class.new}
+  let(:lunch_desserts) {{ Desserts: 'Price', Pannacotta: 7, Semi_fresso: 7, Tiramisu: 10}}
+  let(:dinner_starters) {{ Starters: 'Price', Calamari: 8, Olives: 4, Panzerottini: 7, Bruschetta: 6, Gamberi: 9, Insalata: 7 }}
+  describe "#Initialize" do
+
+    it {is_expected.to respond_to(:meal)}
+
+    it {is_expected.to respond_to(:course)}
+
+    it 'has dinner as default meal option' do
+      expect(italian_menu.meal).to eq :dinner
+    end
+
+    it 'has all as default course option' do
+      expect(italian_menu.course).to eq :all
+    end
+  end
+
+  describe "#dishes" do
+
+    it 'always returns a hash of dishes and prices' do
+      expect(italian_menu.dishes).to be_a Hash
+    end
+
+    it 'returns dishes for the meal specified in menu class - default dinner/all' do
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)